ui/apps/platform/src/Containers/Collections/CollectionForm.tsx

```diff
@@ -46,6 +51,34 @@ import CollectionAttacher from './CollectionAttacher';
 import CollectionResults from './CollectionResults';
 import { Collection, ScopedResourceSelector, SelectorEntityType } from './types';
 
+function AttachedCollectionTable({ collections }: { collections: CollectionResponse[] }) {
+    return collections.length > 0 ? (
+        <TableComposable aria-label="Attached collections" variant={TableVariant.compact}>
+            <Tbody>
+                {collections.map(({ name, description }) => (
+                    <Tr key={name}>
+                        <Td dataLabel="Name">
+                            <Button variant="link" className="pf-u-pl-0" isInline>
+                                {name}
+                            </Button>
+                        </Td>
+                        <Td dataLabel="Description">
+                            <Truncate content={description} />
+                        </Td>
+                    </Tr>
+                ))}
+            </Tbody>
+        </TableComposable>
+    ) : (
+        <EmptyState variant={EmptyStateVariant.xs}>
+            <EmptyStateIcon icon={CubesIcon} />
+            <Title headingLevel="h4">
+                There are no other collections attached to this collection
+            </Title>
+        </EmptyState>
```

Comment:
This is consistent with many other empty state titles. However, a question for PatternFly Developer Office Hours is pro and con for accessibility of `h4` element versus `p` element with the identical style.

In a video for a past day of learning with theme of accessibility, the screen reader user used heading structure of page quite a bit to form a mental model of the information hierarchy (and commented on any skips in heading levels). This page has intuitive hierarchy for `h1` and `h2` but not so much for this heading.

Reply:
True I agree, especially with the screen reader comment since this bit of text doesn't really communicate hierarchy. The page has `h1` and `h2`, but there isn't anything semantically that makes this a `h3`, `h4`, etc. I'll update to a `<p>` with styles.